src/java/main/org/apache/zookeeper/server/quorum/QuorumCnxManager.java

```diff
@@ -233,25 +233,25 @@ public QuorumCnxManager(QuorumPeer self) {
         listener = new Listener();
         listener.setName("QuorumPeerListener");
     }
-
+    
```

Comment:
I cannot help doing this although I know it will increase review work.I will keep it in my mind next time. **"When we say "do it separately" in the context of zk, it probably implies 'never'"** Quotation from hanm.